src/TurbulenceClosures/turbulence_closure_implementations/scalar_biharmonic_diffusivity.jl

```diff
@@ -117,3 +120,25 @@ function on_architecture(to, closure::ScalarBiharmonicDiffusivity{F, <:Any, <:An
     κ = on_architecture(to, closure.κ)
     return ScalarBiharmonicDiffusivity{F, N}(ν, κ)
 end
+
+# We do not have at the moment the ability to distinguish between the halos required in the x, y and z
+# direction for a general closure since the closure could contain any user-defined function
+
+# In the end we probably want something like this:
+#= 
+required_halo_size_x(::ScalarBiharmonicDiffusivity{<:Any, ThreeDimensionalFormulation, N}) where N = N
+required_halo_size_y(::ScalarBiharmonicDiffusivity{<:Any, ThreeDimensionalFormulation, N}) where N = N
+required_halo_size_z(::ScalarBiharmonicDiffusivity{<:Any, ThreeDimensionalFormulation, N}) where N = N
```

Comment:
This code may go stale cause there's no way to test it so we may not want to include a comment like this